lnrpc/walletrpc/walletkit_server_test.go

```diff
@@ -0,0 +1,59 @@
+//go:build walletrpc
+// +build walletrpc
+
+package walletrpc
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/lightningnetwork/lnd/input"
+	"github.com/stretchr/testify/require"
+)
+
+// TestWitnessTypeMappings tests that the mapping between the witness type
+// enum defined in the `input` package and the witness type enum protocol
+// buffer definition in the `walletrpc` package is correct.
+func TestWitnessTypeMapping(t *testing.T) {
+	t.Parallel()
+
+	require.Equal(
```

Comment:
maybe add the comment : test that both enum types have the same length except the `UNKNOWN_WITNESS` type which is only present in the walletrpc witness type enum?